view/template/acquisition/_youtube_error.php

```diff
@@ -0,0 +1,5 @@
+<?php
+
+$error_message = AcquisitionActions::getYoutubeErrorMessage($error_message ,$email);
```

Comment:
Stop breaking MVC! Use `prepareXXX` method. (Code search for "prepare" in an action to figure this pattern out)